app/javascript/chat/chat.jsx

```diff
@@ -52,12 +53,14 @@ export default class Chat extends Component {
     chatChannels: PropTypes.string.isRequired,
     chatOptions: PropTypes.string.isRequired,
     githubToken: PropTypes.string.isRequired,
+    tagModerator: PropTypes.object.isRequired,
```

Comment:
refactor (blocking): Use [PropTypes.shape](https://reactjs.org/docs/typechecking-with-proptypes.html) instead of the generic `PropTypes.object`.

```suggestion
    tagModerator: PropTypes.shape({ isTagModerator: PropTypes.bool.isRequired }).isRequired,
```